editor/import/resource_importer_csv_translation.cpp

```diff
@@ -142,13 +142,16 @@ Error ResourceImporterCSVTranslation::import(ResourceUID::ID p_source_id, const
 		}
 
 		String save_path = p_source_file.get_basename() + "." + translations[i]->get_locale() + ".translation";
+		ResourceUID::ID save_id = hash64_murmur3_64(translations[i]->get_locale().hash64(), p_source_id) & 0x7FFFFFFFFFFFFFFF;
+		if (ResourceUID::get_singleton()->has_id(save_id)) {
+			save_path = ResourceUID::get_singleton()->get_id_path(save_id);
```

Comment:
you will still assign this uid below : `ResourceSaver::set_uid(save_path, save_id);` even if it already exists. That seems like a problem (now 2 different resources in the project have the same uid) ?

Reply:
This code assumes that the existing UID was associated with old path (before move/rename), so the UID will still point to single path.
But yeah, `set_uid()` is redundant if it already exists.

Reply:
I see. But if this is a real collision (very unlikely, but it could happen), you will then destroy another resource silently. I don't know if we can catch that somehow.